backend/case/src/main/java/org/pucar/dristi/enrichment/CaseRegistrationEnrichment.java

```diff
@@ -235,15 +235,23 @@ public void enrichCaseApplicationUponUpdate(CaseRequest caseRequest) {
             throw new CustomException(ENRICHMENT_EXCEPTION, "Error in case enrichment service during case update process: " + e.getMessage());
         }
     }
-    public void enrichCaseNumberAndCNRNumber(CaseRequest caseRequest) {
+    public void enrichCaseNumberAndCourtCaseNumber(CaseRequest caseRequest) {
         try {
             List<String> courtCaseRegistrationCaseNumberIdList = idgenUtil.getIdList(caseRequest.getRequestInfo(), caseRequest.getCases().getTenantId(), config.getCaseNumberCc(), null, 1);
             caseRequest.getCases().setCaseNumber(courtCaseRegistrationCaseNumberIdList.get(0));
             caseRequest.getCases().setCourtCaseNumber(courtCaseRegistrationCaseNumberIdList.get(0));
-            caseRequest.getCases().setCnrNumber(caseUtil.getCNRNumber(caseRequest.getCases().getFilingNumber(), STATE, DISTRICT, ESTABLISHMENT_CODE));
         } catch (Exception e) {
```

Comment:
CNR number generation should use CourtID as a prefix directly rather than using State, District and Establishment.

Reply:
What is CaseNumber? A case is supposed to have 

filingNumber
cmpNumber
cnrNumber
courtCaseNumber (CC/ST number)

Reply:
line 243 is removed. also I will remove the caseNumber